src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -58,11 +58,22 @@ const render = function render(interaction) {
 
         const placeholderText = interaction.attr('placeholderText');
 
+        const getItemLanguage = function getItemLanguage() {
```

Comment:
**nitpick:** This could be an arrow function

**suggestion:**
```suggestion
        const getItemLanguage = () => {
```

Reply:
ok!